src/index.js

```diff
@@ -15,6 +15,7 @@ export default (
     branch,
     bail,
     check,
+    ignorePath = '.prettierignore',
```

Comment:
Should still work if you just do this? (As the `undefined` value will get overwritten inside `createIgnorer`.)
```suggestion
    ignorePath,
```